lib/timeline/component/timeline-chart/TimelineChartLineGraph.js

```diff
@@ -173,6 +193,50 @@ class TimelineChartLineGraph extends LineGraph {
     return false;
   }
 
+  setOptions(options) {

```

Comment:
Is this **setOptions** method needed to do some specific behavior?

Reply:
Yes. When the componente is registered (TimelineChart.js:84), the parent class read the variable this.componentes and invoke some methods. One of them is the setOptions. The Core.js pass automatically the options through the componente and the setOptions filter only the properties specified at fields (TimelineChartLineGraph.js:198).